api.go

```diff
@@ -474,45 +474,45 @@ func DeleteContainer(podID, containerID string) (*Container, error) {
 
 // StartContainer is the virtcontainers container starting entry point.
 // StartContainer starts an already created container.
-func StartContainer(podID, containerID string) (*Container, error) {
+func StartContainer(podID, containerID string) (*Pod, *Container, error) {
```

Comment:
I'd prefer if we keep the `StartContainer()` prototype and provide an additional `pod.go:PodFromID()` getter instead.